src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -175,4 +182,33 @@ private ProjectBag gather(Project project) {
         return new ProjectBag(projects, envelopes);
     }
 
+    public Page<Project> filterProjects(String search, String wranglingState, String wrangler, Pageable pageable) {
```

Comment:
I would create a Pojo class for the search parameters. Something like:

```java
class SearchFilter {
   String search;
   String wranglingState;
   String wrangler;
}
```

then, change the signature of the relevant controller & service methods.
This way, when we add options to the search, we need only to change the search filer class and implementation.